cfme/tests/containers/test_containers_smartstate_analysis.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import paginator, toolbar as tb, CheckboxTable, flash
+from utils import testgen
+from utils.version import current_version
+from cfme.web_ui import InfoBlock
+from cfme.configure import tasks
+import re
+from random import choice
+from utils.ssh import SSHTail
+
+# CMP - 9496
+pytestmark = [
+    pytest.mark.meta(server_roles='+smartproxy'),
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+tasks_tbl = CheckboxTable(table_locator="//table[@class='table table-striped" +
```

Comment:
The `+` operator here is not necessary; python will concatenate the strings even without it, as long as the whole string is wrapped in parentheses.